src/fiatconnect/KycLanding.tsx

```diff
@@ -12,7 +14,20 @@ export interface Props {
 }
 
 function KycLanding(_props: Props) {
```

Comment:
It might make the code slightly cleaner to destructure quote and personaKycStatus from the props here. It will avoid so many `prop.` in the rest of the component.